content/departments/cloud/technical-docs/index.md

```diff
@@ -62,7 +62,7 @@ Sourcegraph upgrades every test and customer instances according to [SLA](#slas-
 The release process is performed in steps:
 
 1. New version is released via [release guild](../../engineering/dev/process/releases/release_guild.md)
-2. Github issue in [Sourcegraph repository](https://github.com/sourcegraph/sourcegraph) is open based on the [managed instances upgrade template](../../engineering/dev/process/releases/upgrade_managed_issue_template.md)
+2. Github issue in [Sourcegraph repository](https://github.com/sourcegraph/sourcegraph) is open based on the managed instances upgrade template.
```

Comment:
```suggestion
2. GitHub issue in [Sourcegraph repository](https://github.com/sourcegraph/sourcegraph) is open based on the managed instances upgrade template.
```

Since you've touched this line